scripts/compile_bundle.sh

```diff
@@ -153,31 +156,25 @@ unset LLVM_DIR
 unset USE_LLVM
 python -m pip uninstall -y mkl-static mkl-include
 python -m pip install --force-reinstall dist/*.whl
-git config --global --unset safe.directory
 #  TorchVision
 cd ../vision
-git config --global --add safe.directory `pwd`
-conda install -y libpng jpeg
 python setup.py clean
 python setup.py bdist_wheel 2>&1 | tee build.log
 python -m pip install --force-reinstall --no-deps dist/*.whl
-python -m pip install Pillow
-git config --global --unset safe.directory
+# don't fail on external scripts
+set +uex
+source ${DPCPP_ENV}
+source ${ONEMKL_ENV}
+set -uex
```

Comment:
Does torchaudio compile with mkl-static? if so, please move line 157 `python -m pip uninstall -y mkl-static mkl-include` to the end of audio compilation.

Reply:
torch audio doesn't seem to build against python mkl-static? the mkl requirement seems to come from the `third_party/kaldi/` sub-module, not audio itself.

Setting LD_LIBRARY_PATH, MKLROOT or CMAKE_PREFIX_PATH to the path where python mkl-static is installed, doesn't seem to help. 

```
FAILED: torchaudio/csrc/libtorchaudio.so
[...]
/usr/bin/ld: cannot find -lmkl_intel_ilp64: No such file or directory
/usr/bin/ld: cannot find -lmkl_intel_thread: No such file or directory
/usr/bin/ld: cannot find -lmkl_core: No such file or directory
collect2: error: ld returned 1 exit status
```
not sure how to proceed here

Reply:
mkl requirement come from the `third_party/kaldi`/ sub-module

i am using virtualenv, not sure if this happens with conda.

Reply:
Let's source MKL before compiling torchaudio then.